lib/Config.py

```diff
@@ -82,3 +84,19 @@ def getFile(cls, getfile, unpack=True):
           if len(fzip.namelist())>0:
             data=BytesIO(fzip.read(fzip.namelist()[0]))
     return (data, response)
+  
+  # Get Microsoft Bulletin
```

Comment:
Why would this be in the Config? Would it be rather in the module dedicated to MSFT bulletins?